amanuensis/auth/__init__.py

```diff
@@ -0,0 +1,45 @@
+from os import environ
+import binascii
+from cdislogging import get_logger
+from amanuensis.auth.errors import NoPrivateKeyError
+from Crypto.PublicKey import RSA
+from Crypto.Signature import pkcs1_15
+from Crypto.Hash import SHA256
+
+
+
+logger = get_logger(__name__)
+
+
+def loadkey():
+    """
+    Load private key
+    """
+    priv_key = None
+    private_key_path = environ.get("PRIVATE_KEY_PATH", None)
```

Comment:
could you move this to the config instead of the ENV. Same reason as Fence, all the other configs are in the config file so we can keep everything in the same place